components/Typography/tests/unit/UIFont+MaterialScalableTests.m

```diff
@@ -0,0 +1,148 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+#import "MaterialTypography.h"
+
+@interface UIFont_MaterialScalableTests : XCTestCase
+
+@end
+
+@implementation UIFont_MaterialScalableTests
+
```

Comment:
Please add at least one test that doesn't test the behavior of MDCFontScaler. For example, 

1. A test for the fall-back behavior when a scale isn't set for `.AccessibilityMedium` or above.
2. Tests for custom curves that do arbitrary things (not necessarily increasing).
3. Test for a non-scaled font returning itself.
4. Test for scales with negative values return the original font
5. Override `mdc_safeSharedApplication` to return an expected ContentSizeCategory and verify the "current" scaling behavior matches an expected curve.

Reply:
1. Done.
2. Values MUST NOT decrease.  If you do encode something like that, you are probably a bad person and should be shunned ;)
3. Done.
4. Done for negative and zero.
5. This is a good idea, but should be handled as a bigger set of tests against this new and our
existing Dynamic Type scaling.